lib/flight_job/command.rb

```diff
@@ -203,8 +203,15 @@ def load_job(id)
     def assert_results_dir_exists(job)
       # NOTE: Jobs created with old versions of flight-job will not have a
       # results directory.
-      unless job.results_dir
-        raise MissingError, "The job did not report its results directory"
+      if !(job.results_dir && Dir.exists?(job.results_dir))
+        case job.state
+        when 'PENDING'
+          raise MissingError, 'Your job has not started, please try again latter...'
```

Comment:
`latter` should be spelled `later`.  Here and elsewhere please.

Reply:
57a3f08de3ee67b0da9e0992f505c22fad679b9b